src/components/ClusterManager.js

```diff
@@ -165,6 +167,10 @@ export default class ClusterManager extends PureComponent {
     } else if (cluster.status === 'Running' && prevCluster.status === 'Updating') {
       notify('success', 'Number of workers has updated successfully.')
     }
+    if (prevApp && prevApp.status !== 'RUNNING' && app && app.status === 'RUNNING') {
```

Comment:
Do we want to check `not RUNNING` or `is PROVISIONING`? (I don't think there's a functional difference right now, but it is a difference in behavior)

Reply:
I think this best captures the intended logic, that we want to catch the transition to `RUNNING` status.